feat: add new worklow to track the voting process (#1278)
asyncapi/community: voting.md

The Git Vote bot adds a comment that voting is completed.

### Tracking Voting 

> We store [the voting history of entire TSC](voteTrackingDetail.md) since voting automation was introduces.

Reviewer: ```suggestion
We store [the voting history of entire TSC](TSC_VOTING_OVERVIEW.md) since voting automation was introduces.
```

> We store [the voting history of entire TSC](https://github.com/asyncapi/community/blob/master/voteTrackingDetail.md) since voting automation was introduces.

Reviewer: ```suggestion
We store [the voting history of entire TSC](voteTrackingDetail.md) since voting automation was introduces.
```

Reviewer: and since we have real historical data, you can also push `voteTrackingDetail.md` to this PR

Reviewer: and please make sure that the markdown document have a [hidden html comment](https://www.tutorialspoint.com/How-to-create-hidden-comments-in-HTML#:~:text=To%20create%20hidden%20comments%20in%20HTML%2C%20we%20add%20%3C!,this%20tag%20it%20is%20hidden.) that explains the file is generated, and should not be manually updated - unless there is a visible mistake + point to the script that is responsible for updating the document

> The [voteTrackingDetail](https://github.com/asyncapi/community/blob/master/voteTrackingDetail.md) file contains the complete voting history of all TSC Members, whether they voted or not in the community's voting processes.

Reviewer: ```suggestion
We store [the voting history of entire TSC](https://github.com/asyncapi/community/blob/master/voteTrackingDetail.md) since voting automation was introduces.
```

> * 🔕 indicates that the member did not vote in the last voting.
> * 👍 indicates that the member was in favor in the last voting.
> * 👎 indicates that the member was against in the last voting.
> * 👀 indicates that the member abstained in the last voting.

Reviewer: ```suggestion
* 🔕 indicates that the member did not vote.
* 👍 indicates that the member was in favor.
* 👎 indicates that the member was against.
* 👀 indicates that the member abstained from voting.
```

> Tracking file us automatically updated by a bot after each vote is completed. It also records whether the member has voted in the last 3 months, helping to identify inactive TSC Members.

Reviewer: ```suggestion
Tracking file is automatically updated by a bot after each vote is completed. It also records whether the member has voted in the last 3 months, helping to identify inactive TSC Members.
```

> The voteTrackingDetails file also tracks whether the member has voted in the last 3 months, helping to identify inactive TSC Members.

Reviewer: ```suggestion
Tracking file us automatically updated by a bot after each vote is completed. It also records whether the member has voted in the last 3 months, helping to identify inactive TSC Members.
```
